lib/charms/grafana_k8s/v0/grafana_source.py

```diff
@@ -97,9 +98,8 @@ def __init__(
         self._relation_name = relation_name
         self.expected_relation_interface = expected_relation_role
         self.actual_relation_role = actual_relation_role
-        self.message = (
-            f"The '{relation_name}' relation has role '{repr(actual_relation_role)}' "
-            f"rather than the expected '{repr(expected_relation_role)}'"
+        self.message = "The '{}' relation has role '{}' " "rather than the expected '{}'".format(
```

Comment:
```suggestion
        self.message = "The '{}' relation has role '{}' rather than the expected '{}'".format(
```
if fmt is gonna insist on it being on the same line, there's no need to split the string 😄

Reply:
Bad `black` :/